.spectral.yml

```
@@ -64,7 +64,7 @@ rules:
       functionOptions:
         schema:
           type: string
-          pattern: '^[0-9]+\.[0-9]+\.[0-9]+$'
+          pattern: '^[0-9]+\.[0-9]+\.[0-9]+[a-zA-Z-]*$'
```

This would also allow e.g. `1.0.0Foo`. Better:
```suggestion
          pattern: '^[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z-]+)?$'
```
or even
```suggestion
          pattern: '^[0-9]+\.[0-9]+\.[0-9]+(-[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?$'
```
to fully match the semver pre-release spec and allow e.g. `1.0.0-rc.1` (https://semver.org/spec/v2.0.0-rc.1.html#spec-item-10)